mintapi/api.py

```diff
@@ -459,6 +472,87 @@ def __x_months_ago(self, months=2):
             day=1
         )
 
+    def get_trend_data(
+        self,
+        report_type: ReportView.Options,
+        date_filter: DateFilter.Options,
+        start_date: Optional[str] = None,
+        end_date: Optional[str] = None,
+        category_ids: List[str] = None,
+        tag_ids: List[str] = None,
+        descriptions: List[str] = None,
+        match_all_filters: bool = True,
+        limit: int = 1000,
+        offset: int = 0,
+    ) -> List[Dict]:
+        """
+        Public accessor for trend data. Internally constructs a trend api payload
+
+
+        Parameters
```

Comment:
Can we add this in some capacity to the README?  In addition, because this endpoint involves so much more than the others, it'd be good to have an example or two in the README.